include/wsrep/connection_monitor_service.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2024-2026 Codership Oy <info@codership.com>
+ *
+ * This file is part of wsrep-lib.
+ *
+ * Wsrep-lib is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Wsrep-lib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with wsrep-lib.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+/** @file connection_monitor_service.hpp
+ *
+ * Service interface for interacting with DBMS provided
+ * connection monitor callback.
+ */
+
+#ifndef WSREP_CONNECTION_MONITOR_SERVICE_HPP
+#define WSREP_CONNECTION_MONITOR_SERVICE_HPP
+
+#include "compiler.hpp"
+#include "wsrep/buffer.hpp"
+#include "v26/wsrep_connection_monitor_service.h"
```

Comment:
Wsrep-API headers should not be exposed in wsrep-lib public headers. If you need this header for `wsrep_connection_key_t` type, I guess doing a matching typedef in this header would suffice.

Reply:
Added typedef